coverage/cmdline.py

```diff
@@ -127,6 +127,11 @@ class Opts:
         '', '--pretty-print', action='store_true',
         help="Format the JSON for human readers.",
     )
+    lcov = optparse.make_option(
+        '-l', '--lcov', action='store', dest='outfile',
```

Comment:
This should be `-o --output` I think.  There's an earlier PR that used `-l`, but that was hacking it into an existing command.